internal/terraform/context_apply2_test.go

```diff
@@ -912,3 +913,140 @@ resource "test_resource" "c" {
 		}
 	})
 }
+
+// pass an input through some expanded values, and back to a provider to make
+// sure we can fully evaluate a provider configuration during a destroy plan.
+func TestContext2Apply_destroyWithConfiguredProvider(t *testing.T) {
+	m := testModuleInline(t, map[string]string{
+		"main.tf": `
+variable "in" {
+  type = map(string)
+  default = {
+    "a" = "first"
+	"b" = "second"
```

Comment:
FWIW, I've typically just used Go-style tab-based indentation for situations where we have configuration embedded in raw strings inside Go code, because it avoids this sort of editor fighting and isn't really important unless the test is actually testing something related to formatting.

So making this all be tabs would also have been an acceptable answer as far as I'm concerned; it was the inconsistency that bothered me. :grinning: